recipes/emd-signal/LICENSE.md

```diff
@@ -0,0 +1,190 @@
+                                 Apache License
```

Comment:
This file is in the tarball, no need to add it here again. Conda will use that one.